cmake/Intel.cmake

```diff
@@ -43,3 +43,9 @@ if(APPLE)
   # Passing the -c flag includes them and fixes an error with undefined symbols
   set(CMAKE_Fortran_ARCHIVE_FINISH "<CMAKE_RANLIB> -c <TARGET>")
 endif()
+
+# This must be last, to override all other optimization settings.
+if(DISABLE_FMA)
+  set(CMAKE_Fortran_FLAGS_RELEASE "${CMAKE_Fortran_FLAGS_RELEASE} -no-fma")
+  set(CMAKE_C_FLAGS_RELEASE "${CMAKE_Fortran_FLAGS_RELEASE} -no-fma")
+endif()
```

Comment:
@SamuelTrahanNOAA Could we include the -no-fma option inside the FASTER section so we don't need to add a special "DISABLE_FMA" in CMakeLists ?

Reply:
No. This is just a workaround for WCOSS2 and Acorn. The platform-specific options are not in Intel.cmake, so there must be another flag to pass -no-fma in.

Reply:
We have CMAKE_Platform variable defined. You can use it to identify WCOSS2 and Acorn platforms and set appropriate flags.

Reply:
That's not how other parts of the build work. The design of the build system is for platform-specific settings to be in `cmake/configure_platform.compiler.cmake`. Breaking the design just to eliminate one `-D` option is worse than having the `-D`

Also, that option is not *only* needed for the WCOSS2 and Acorn systems. It is needed for other AMD-based systems as well. It just so happens that WCOSS2 and Acorn have the combination of compiler and chip that cause this to break.

If you hide the workaround from users that may need it, not only are you breaking the design, but you're also making the platform porting experience objectively worse.